README.md

```diff
@@ -1 +1,63 @@
-# Hello-World
\ No newline at end of file
+
+# Hello World
+
+In this project, I set up a "Hello world" repository. No complex coding is required for this exercise. My goal is to master all of the tools and best practices I learnt about in previous steps. 
+
+
+## Built With
+
+- Html
+- css
+
+
+## Live Demo (if available)
+
+None
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple  steps.
+
+ Prerequisites
+
+ Setup
+
+ Install
+
+ Use
+
+ Run tests
+
+ Deploy
+
+
+
+## Authors
+
+👤 **jacinta Ngwu**
+
+- GitHub: [@jacintangwu](https://github.com/JacintaNgwu/Hello-World.git)
+- Twitter: [@jacinta_ngwu](https://twitter.com/jacinta_ngwu)
+
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check for issues and contribute.
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+
+## 📝 License
+
+None
```

Comment:
- [ ] Credit goes to the MIT, I suggest you to add `This project is [MIT](https://github.com/LICENSE) licensed.`